app/support/products/scheduling_support.rb

```diff
@@ -108,6 +112,10 @@ def online?
 
   private
 
+  def current_offline_reservation
+    offline_reservations.current.last
```

Comment:
There ought to be only one current offline reservation for an Instrument. It could be `.first` even.